### ISSUE TITLE
Exception NoClassDefFoundError version 193.8.1

### ISSUE THREAD

**Issue by rafamoreira** [zielu/GitToolBox#237]
Hello, I don't know what the problem is, I tried install from the marketplace inside the IDE, and manually to no avail.

Some things are working, but for example the list of the files on the project tab, stay forever on loading ...

OS: Linux
IDE: Rubymine

Here's the stacktrace

```
java.lang.NoClassDefFoundError: com/intellij/ide/projectView/impl/nodes/PackageElementNode
	at zielu.gittoolbox.ui.projectview.GitRepositoryFinder.isChildOfViewModuleNode(GitRepositoryFinder.kt:84)
	at zielu.gittoolbox.ui.projectview.GitRepositoryFinder.getRepoFor(GitRepositoryFinder.kt:38)
	at zielu.gittoolbox.ui.projectview.ProjectViewDecorator.lambda$doDecorate$1(ProjectViewDecorator.java:39)
	at com.codahale.metrics.Timer.timeSupplier(Timer.java:136)
	at zielu.gittoolbox.ui.projectview.ProjectViewDecorator.doDecorate(ProjectViewDecorator.java:39)
	at zielu.gittoolbox.ui.projectview.ProjectViewDecorator.lambda$decorate$0(ProjectViewDecorator.java:27)
	at com.codahale.metrics.Timer.time(Timer.java:151)
	at zielu.gittoolbox.ui.projectview.ProjectViewDecorator.decorate(ProjectViewDecorator.java:27)
	at com.intellij.ide.projectView.impl.CompoundProjectViewNodeDecorator.lambda$decorate$0(CompoundProjectViewNodeDecorator.java:49)
	at com.intellij.ide.projectView.impl.CompoundProjectViewNodeDecorator.forEach(CompoundProjectViewNodeDecorator.java:60)
	at com.intellij.ide.projectView.impl.CompoundProjectViewNodeDecorator.decorate(CompoundProjectViewNodeDecorator.java:49)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.lambda$doUpdate$1(AbstractPsiBasedNode.java:166)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:791)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.doUpdate(AbstractPsiBasedNode.java:139)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.lambda$update$0(AbstractPsiBasedNode.java:135)
	at com.intellij.util.AstLoadingFilter.lambda$toComputable$2(AstLoadingFilter.java:167)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:125)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:114)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:109)
	at com.intellij.ide.projectView.impl.nodes.AbstractPsiBasedNode.update(AbstractPsiBasedNode.java:135)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.getUpdatedPresentation(PresentableNodeDescriptor.java:76)
	at com.intellij.ide.util.treeView.PresentableNodeDescriptor.update(PresentableNodeDescriptor.java:28)
	at com.intellij.ui.tree.StructureTreeModel$Node.update(StructureTreeModel.java:451)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:433)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:425)
	at com.intellij.ui.tree.StructureTreeModel$Node.<init>(StructureTreeModel.java:419)
	at com.intellij.ui.tree.StructureTreeModel.getValidChildren(StructureTreeModel.java:389)
	at com.intellij.ui.tree.StructureTreeModel.validateChildren(StructureTreeModel.java:300)
	at com.intellij.ui.tree.StructureTreeModel.getNode(StructureTreeModel.java:294)
	at com.intellij.ui.tree.StructureTreeModel.getChildren(StructureTreeModel.java:314)
	at com.intellij.ui.tree.AsyncTreeModel$CmdGetChildren.getNode(AsyncTreeModel.java:586)
	at com.intellij.ui.tree.AsyncTreeModel$ObsolescentCommand.get(AsyncTreeModel.java:481)
	at com.intellij.ui.tree.AsyncTreeModel$ObsolescentCommand.get(AsyncTreeModel.java:447)
	at com.intellij.util.concurrency.Command$Processor.lambda$process$1(Command.java:48)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:997)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:73)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:127)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:116)
	at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:73)
	at com.intellij.util.concurrency.Invoker.invokeSafely(Invoker.java:191)
	at com.intellij.util.concurrency.Invoker.lambda$offerSafely$0(Invoker.java:158)
	at com.intellij.util.concurrency.Invoker$Background.lambda$offer$0(Invoker.java:473)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:217)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:30)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:190)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.ClassNotFoundException: com.intellij.ide.projectView.impl.nodes.PackageElementNode PluginClassLoader[zielu.gittoolbox, 193.8.1] com.intellij.ide.plugins.cl.PluginClassLoader@1ada7579
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:77)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 58 more
```

**Comment by zielu** [comment]
Hi, sorry for that will fix it asap

**Comment by zielu** [comment]
Fixed in 193.8.2
